### PR TITLE
rkt: new resource agent for managing containers

### PR DESCRIPTION
**Pull request by vvidic**
Based on a similar docker resource agent.

### CODE REVIEW

File: heartbeat/rkt [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+provided in the 'name' argument of this agent.
+
+</longdesc>
+<shortdesc lang="en">run options</shortdesc>
+<content type="string"/>
+</parameter>
+
+<parameter name="run_cmd" required="0" unique="0">
+<longdesc lang="en">
+Specifiy a command to launch within the container once
```
Comment (jnpkrn) [inline heartbeat/rkt]: typo

Comment (vvidic) [reply heartbeat/rkt]: Tnx, updated. Note that docker agent has the same "Specifiy" typo :)

**Comment by krig** [comment]
@vvidic Could you also add the agent to the list in `Makefile.am`? That way it gets checked by Travis CI and the man page gets built etc.

**Comment by vvidic** [comment]
ACK, branch updated. Should I also fix these warnings reported in the Travis build?

```
[WARNING] Checking heartbeat/rkt...
heartbeat/rkt:33:3: note: Double quote to prevent globbing and word splitting. [SC2086]
heartbeat/rkt:34:3: note: Double quote to prevent globbing and word splitting. [SC2086]
heartbeat/rkt:176:10: note: Double quote to prevent globbing and word splitting. [SC2086]
heartbeat/rkt:180:18: note: Double quote to prevent globbing and word splitting. [SC2086]
heartbeat/rkt:180:24: note: Double quote to prevent globbing and word splitting. [SC2086]
heartbeat/rkt:187:8: note: Double quote to prevent globbing and word splitting. [SC2086]
heartbeat/rkt:195:9: note: Double quote to prevent globbing and word splitting. [SC2086]
heartbeat/rkt:200:34: note: Double quote to prevent globbing and word splitting. [SC2086]
heartbeat/rkt:205:41: note: Double quote to prevent globbing and word splitting. [SC2086]
heartbeat/rkt:210:34: note: Double quote to prevent globbing and word splitting. [SC2086]
heartbeat/rkt:218:7: note: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?. [SC2181]
...
```

**Comment by krig** [comment]
I usually ignore the double quoting warnings unless I can see that there's a real risk (like user-provided filenames for example). The last one seems a bit pedantic. Description / motivation here:

https://github.com/koalaman/shellcheck/wiki/SC2181

**Comment by krig** [comment]
Alright, since it passes validation and human review and it's a new agent, I don't see any reason not to merge.

**Pull request merged**